test/nn/encoding/test_cyclic_encoding.py

```diff
@@ -0,0 +1,24 @@
+import torch
+
+from torch_frame.nn import CyclicEncoding
+
+
+def test_cyclic_encoding_shape():
+    out_size = 8
+    for size in [(10, ), (10, 4), (10, 5, 8)]:
+        input_tensor = torch.rand(size)
+        cyclic_encoding = CyclicEncoding(out_size)
+        out_tensor = cyclic_encoding(input_tensor)
+        assert out_tensor.shape == input_tensor.shape + (out_size, )
+
+
+def test_cyclic_encoding_values():
+    out_size = 8
+    for size in [(10, ), (10, 4), (10, 5, 8)]:
+        input_tensor = torch.randint(0, 2, size=size)
```

Comment:
Can we generate fload input tensor?
Can we separately test 
```python
input_zeros_tensor = torch.zeros(size)
input_ones_tensor = torch.ones(size)
```
Gave exactly the same encoded output?